### PR TITLE
[release-1.5] Add note on automatic mutual TLS in the upgrade note

### PR DESCRIPTION
**Pull request by johnma14**

### CODE REVIEW

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md [hunk 1]
```diff
@@ -74,6 +74,10 @@ Please check [Mixer Deprecation](https://tinyurl.com/mixer-deprecation) notice f
 * We have stabilized the SDS certificate and key provisioning flow. Now the Istio workloads are using SDS to provision certificates. The secret volume mount approach is deprecated.
 * Please note when mutual TLS is enabled, Prometheus deployment needs to be manually modified to monitor the workloads. The details are described in this [issue](https://github.com/istio/istio/issues/21843). This is not required in 1.5.1.
 
+## Automatic mutual TLS
+ 
+Automatic mutual TLS is now enabled by default. Traffic between sidecars are automatically configured as mutual TLS. You can disable this explicitly if you worry about the encryption overhead by adding the option `-- set values.global.mtls.auto=false` during install. For more details, check the [automatic mutual TLS task](/docs/tasks/security/authentication/auto-mtls/)
```
Comment (incfly) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: miss a "."

Comment (incfly) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: question for native speakers, traffic should be uncountable noun, thus "s/are/is"?

**Comment by ericvn** [comment]
I did Slack Mariam to fix the few lint errors here and about the link change. Waiting on a commit for that.

File: content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md (hunk 1 above)
Comment (ericvn) [reply content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]: You are correct it should be `is`.

Comment (frankbu) [inline content/en/news/releases/1.5.x/announcing-1.5/upgrade-notes/index.md]:
```suggestion
Automatic mutual TLS is now enabled by default. Traffic between sidecars is automatically configured as mutual TLS. You can disable this explicitly if you worry about the encryption overhead by adding the option `-- set values.global.mtls.auto=false` during install. For more details, refer to [automatic mutual TLS](/docs/tasks/security/authentication/authn-policy/#auto-mutual-tls).
```

**Pull request merged**